dataprofiler/profilers/float_column_profile.py

```diff
@@ -285,7 +285,7 @@ def _get_float_precision(
         return subset_precision
 
     @classmethod
-    def _is_each_row_float(cls, df_series: pd.Series) -> List[bool]:
+    def _is_each_row_float(cls, df_series: pd.Series[float]) -> List[bool]:
```

Comment:
I think pd.Series actually has to be an object or str
`pd.Series[str]` or `pd.Series[Object]`

Which is the intent of `NumericStatsMixin.is_float` to determine fro that what a is numeric

Reply:
oh I see, I reverted it